pkg/core/consensus/agreement/handler.go

```diff
@@ -189,7 +191,7 @@ func verifyWhole(a message.Agreement) error {
 
 // AggregatePks reconstructs an aggregated BLS public key from a subcommittee.
 func AggregatePks(p *user.Provisioners, subcommittee sortedset.Set) ([]byte, error) {
-	if cfg.Get().Consensus.UseCompressedKeys {
+	if UseCompressedKeys {
```

Comment:
Any reason you rollback this line?

Reply:
This would have been unintentional, if you don't mind to give a double check there isn't other erroneous changes that would be great.